myblog/blog/views.py

```diff
@@ -61,6 +65,18 @@ class PostViewSet(ModelViewSet):
     def perform_create(self, serializer):
         serializer.save(author = self.request.user)
 
+    @action(detail=True)
+    def tagged_count(self):
+        post = self.get_object()
+        count = post.tagged_count
+        return Response({'tagged_count': count})
+
+    @action(detail=True)
+    def last_tag_date(self):
+        post = self.get_object()
+        date = post.last_tag_date
+        return Response({'last_tag_date': date})
+
```

Comment:
The idea is that these properties are returned in get calls for post model, so there's no need for additional actions.

Reply:
Received!